index.src.html

```diff
@@ -290,6 +290,16 @@
     </ul>
   </section>
 
+  <section algorithm="'create a credential' internal method">
+    <dfn for="Credential" method export>\[[Create]](data)</dfn> is called with a
+    {{CredentialCreationData}}, and returns either a {{Credential}}, if one can be created, or
```

Comment:
The proposal in w3c/webauthn#384 is to invert that such that the [`PublicKeyCredential` contains an `AuthenticatorResponse`][1] (which, in the case of `create()` would be an `AuthenticatorAttestationResponse`)

[1]: https://github.com/w3c/webauthn/pull/384/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR394